chacha20/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "chacha20"
+version = "0.1.0"
+authors = ["RustCrypto Developers"]
+license = "MIT OR Apache-2.0"
+description = "Chacha20 Stream Cipher"
+repository = "https://github.com/RustCrypto/stream-ciphers"
+keywords = ["crypto", "stream-cipher", "trait"]
+categories = ["cryptography", "no-std"]
+
+[dependencies]
+block-cipher-trait = "0.6"
+stream-cipher = "0.3"
+salsa20-core = { path = "../salsa20-core" }
```

Comment:
Do you know of a better way of getting this to build that does not involve pushing `salsa20-core` to crates.io? (the same thing also happens in the `salsa20` crate)

Reply:
You can write `salsa20-core = { version = 0.1, path = "../salsa20-core" }`. IIRC it will use version behind `path` first when built as part of workspace.

Reply:
Thanks for the tip, I can at least still test it locally. Added in 897ad6e.